lib/base/string.hpp

```diff
@@ -72,6 +73,15 @@ class String
 
 	operator const std::string&() const;
 
+	/**
+	 * Conversion function to boost::string_view.
+	 *
+	 * This allows using String as the value for HTTP headers in boost::beast::http::basic_fields::set.
+	 *
+	 * @return A boost::string_view representing this string.
+	 */
```

Comment:
Don't we normally put the docs in the definition?

Reply:
There's a docstring for `class String` in this very same file. So if we do, this one is already inconsistent.

Reply:
Consistent? With putting docs where something (e.g. the class itself or https://github.com/Icinga/icinga2/blob/master/lib/remote/apilistener.cpp#L350-L355) is **defined**?

Reply:
Ok, that's a way of arguing making this variant consistent.